conans/test/utils/genconanfile.py

```diff
@@ -182,12 +194,15 @@ def _default_options_line(self):
         return tmp
 
     @property
-    def _build_requires_line(self):
+    def _build_requirements_method(self):
```

Comment:
Why not keeping both?

Reply:
I will change it to include both, I still would like recipes created with ``GenConanfile()`` to be able to define ``build_requires`` as attributes